### ISSUE TITLE
MDX: Generate `index.html` from `app/route/path/index.mdx` instead of `path.html` when using `@hono/vite-ssg`

### ISSUE THREAD

**Issue by p1ass** [honojs/honox#123]
## Motivation

According to [the SSG Helper document](https://hono.dev/helpers/ssg), Hono SSG Helper generates as below:

- `/` -> `./static/index.html`
- `/path` -> `./static/path.html`
- `/path/` -> `./static/path/index.html` ← I want to generate this with HonoX

This rule makes sense, but I can not generate `index.html` with HonoX.

Is there any solution to generate index.html using HonoX?

## Current Behavior

- `/app/route/path.mdx` -> `./dist/path.html`
- `/app/route/path/index.mdx` -> `./dist/path.html`

## Versions

- hono: 4.0.9
- honox: 0.1.4
- @hono/vite-ssg: 0.1.0

**Comment by yusukebe** [comment]
Hi @p1ass 

This may be fixed with the PR: https://github.com/honojs/honox/pull/125, which is included in the new release `v0.1.9`.

You can specify a `trailingSlash` option:

```ts
import { createApp } from 'honox/server'

const app = createApp({
  trailingSlash: true,
})
```

**Comment by p1ass** [comment]
@yusukebe Thank you for implementing my feature request so quickly!
